Source/AliveLibAE/MainMenu.cpp

```diff
@@ -2408,39 +2435,80 @@ signed int MainMenuController::AbeMotions_Update_4D1F50(DWORD input)
     if (sNum_CamSwappers_5C1B66 > 0)
     {
         // Camera is changing - stay in this screen
-        return 0;
+        return MainMenuCams::eNoChange;
     }
 
     // To the game speak screen
     if (input & InputCommands::eUnPause_OrConfirm)
     {
         SFX_Play_46FBA0(SoundEffect::MenuNavigation_52, 35, 400);
-        return 2;
+        return MainMenuCams::eGamespeakCharacterSelectionCam;
     }
 
     // Return to previous screen
     if (input & InputCommands::eBack)
     {
-        return 3;
+        return MainMenuCams::eOptionsCam;
     }
     else
     {
         // No input - stay in this screen
-        return 0;
+        return MainMenuCams::eNoChange;
     }
 }
 
-signed int MainMenuController::tsub_Input_Check_4D49B0(DWORD /*input*/)
+signed int MainMenuController::PSX_Cooperative_Mode_Update_4D49B0(DWORD /*input*/)
 {
-    NOT_IMPLEMENTED();
-    return 0;
-}
+    DWORD held = sInputObject_5BD4E0.field_0_pads[0].field_C_held;
+    if ( held & (InputCommands::eUnPause_OrConfirm | InputCommands::eBack))
+    {
+        return MainMenuCams::eBackstory_Or_NewGameCam;
+    }
 
+    return MainMenuCams::eNoChange;
+}
 
-signed int MainMenuController::tsub_4D48C0(DWORD /*input*/)
+signed int MainMenuController::PSX_Gamemode_Selection_Update_4D48C0(DWORD input)
 {
-    NOT_IMPLEMENTED();
-    return 0;
+    __int16 result; // ax
```

Comment:
you should try to remove this var if possible, looks like you could just use early/in place returns?